cloud/main.js

```diff
@@ -16,7 +16,7 @@ Parse.Cloud.define('sync', function (req, res) {
     return;
   }
 
-  FullContact.asyncContactsAfter(user.get('fullContactAccessToken'), user.get('fullContactLatestCursor')).then(function(fullContacts) {
+  FullContact.asyncContacts(user.get('fullContactAccessToken')).then(function(fullContacts) {
```

Comment:
I'm pretty sure this will cause a full sync every time someone calls this function. I'd rather this be incremental, like store the last known cursor for each user and start there. That way we can limit API calls, since I'll probably be calling this function every N minutes to sync contacts once it's running in production.